README.md

```diff
@@ -48,10 +48,10 @@ Please check out the [issue queue](https://github.com/EddieJaoudeCommunity/suppo
 - **What is a GitHub Organization?** Essentially it's a way to group many repositories in an entity called an organization. An organization also allows us to group people who contribute and are a part of the community. Some examples would be: [Netflix](https://github.com/Netflix), [Facebook](https://github.com/facebook) and of course [Eddie Jaoude Community](https://github.com/EddieJaoudeCommunity).
 Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-organizations).
 - **How do I join the Github Organization?** Raise an issue on this repository asking to join ([examples](https://github.com/EddieJaoudeCommunity/support/issues?q=is%3Aissue+is%3Aclosed+label%3A%22invite+me+to+the+organisation%22)).
-- **How do I set the Organization to Public** You can make the organization public by: 
+- **How do I set the Organization to Public?** You can make the organization public by: 
 	1. Going on to our community page: https://github.com/EddieJaoudeCommunity
-	2. Click on people in this box: <br />
-	![Screenshot_20200916_230238](https://user-images.githubusercontent.com/17693494/93422448-09f0f280-f871-11ea-94d8-c48924a0222e.png)
+	2. Click on the red circled area in this box: <br />
```

Comment:
```suggestion
	2. Click on this area in the box: <br />
```

Reply:
Hi, thank you for your suggestion. I think with the new image, it would be clearer to mention “the red circled area” in this step. Maybe see if other people have any thoughts, then we can take it from there. 🙂

Reply:
Since the next image also has a red circled area, we will have to include this phrase in both sentences and that would look very strange.
I suggest you to instead replace the rounded rectangle with a large oval like below. 
![image](https://user-images.githubusercontent.com/43912470/101637362-ad0d6e80-3a52-11eb-9a7d-b3641281a426.png)
It won't be a problem to identify what `this area` might mean here and also will look consistent.

Do consider. Other opinions welcome too.

Reply:
Just my opinion, but something feels off... perhaps you could highlight one in red and the other in green/blue like the first image? I feel like consistency is important

Reply:
I am sorry, I don't understand. What is it that you find inconsistent?

Reply:
Hi @AkshayaKulasekaran, thank you for your suggestion. I think that's a good idea.

Maybe something like this (see below) for step iii. Is this what you mean?

---

iii. Search your name in the blue circled input :
![screenshot_09122020](https://user-images.githubusercontent.com/13745974/101659458-fbfcd900-3a3d-11eb-84a9-812de0c24fd1.png)

Reply:
I feel that we should keep this PR on Topic and raise another issue for the second image.

Smaller the PR the better 👍

Reply:
@AllanRegush
Agree, thank you. 🙂

Reply:
@AllanRegush, true that
And yes @jasontwuk, that's exactly what I had in mind 😁